deployments/data8/config/common.yaml

```diff
@@ -84,9 +84,9 @@ jupyterhub:
       course::1524699::group::all-admins:
         admin: true
 
-  #    # Data 8, Summer 2023
-  #    # 20230710: commented out in favor of loadRoles above
-  #    course::1525580::enrollment_type::teacher:
-  #      admin: true
-  #    course::1525580::enrollment_type::ta:
-  #      admin: true
+      # Data 8, Summer 2023
```

Comment:
Looks good! Did you mean Spring 2024 here?

You can also delete the YYYYMMDD comment below, and it'd be good to include the github issue number.